clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -90,4 +90,30 @@ public void clear() {
   public void close() {
     store.close();
   }
+
+  // stats
+
+  public long allocatedMemory() {return store.allocatedMemory();}
```

Comment:
Could these be added to a specific stats related interface that `ServerStoreImpl` would implement?

Also, changes to this file caused it to become executable - please revert that.